### ISSUE TITLE
Annotation types/motivations

### ISSUE THREAD

**Issue by mikeapp** [ProjectMirador/mirador-design#7]
It would be interesting to see annotations with `highlighting` and `tagging` motivations represented.

**Comment by aeschylus** [comment]
I agree with this, as they are likely to need different presentations.

**Comment by ggeisler** [comment]
I've had trouble finding examples of manifests that show concrete examples of different motivations. It would be great if we could get some pointers to manifests that show the source of different motivations. Also, this would be a good topic to discuss in a forthcoming Mirador community call.

**Comment by jbhoward-dublin** [comment]
At UCD we have a couple pertinent use cases that may be helpful. 

One involves the classification of graphical graphical components of pre-1700 printed books. These graphical elements can be ornamental letters, illustrations, music, etc.; each is described with metadata, including references to linked data vocabularies. In this case, the 'classifying' motivation applies; if prose descriptions are offered of the iconography of an illustration, then the 'describing' motivation also applies.

Another anticipated used case involves images capturing aspects of Irish folk life from the 1930s through the present. Many of these images would benefit from further identification of places, people, objects (farm implements, household objects and tools, components of boats, etc.), and events; current users of these resources are often in contact with such information. This would represent a use case for the 'identifying' motivation, I think. (Were we able to link an annotation tool to an expansive local folklore vocabulary, classifying would also apply here.)

Finally we frequently have texts with both Irish and English language, the former commonly employing gaelic typefaces. In this case, enabling transcription or translation is desirable, so 'supplementing' is pertinent (possibly also 'transcribing' or 'translating' if appropriate).

**Comment by ggeisler** [comment]
@jbhoward-dublin Thanks for the use case examples, these are great. Hearing about concrete use cases like this is very helpful. 

(And I appreciate your prior comment, which helped me understand why my search for good existing examples of manifests with diverse motivations was difficult and that we should focus on designs that enable users to more easily specify their annotation motivations. When we provide better affordances for specifying motivations, we can expect more manifests to include them and can design the M3 features accordingly.)
